build.sh

```diff
@@ -1,15 +1,15 @@
-#!/bin/bash
+#!/bin/bash -eu
 
-set -eu
+# Make sure we are in the directory this script is in.
+cd "$(dirname "$0")"
 
-if [ ! -d build ]
+if [[ ! -d build ]]
```

Comment:
`[` test command did the same thing.  Not really against this; just pointing it out.